storm-core/src/storm.thrift

```diff
@@ -279,9 +280,10 @@ struct ComponentAggregateStats {
 struct TopologyStats {
 1: optional map<string, i64> window_to_emitted;
 2: optional map<string, i64> window_to_transferred;
-3: optional map<string, double> window_to_complete_latencies_ms;
-4: optional map<string, i64> window_to_acked;
-5: optional map<string, i64> window_to_failed;
+3: optional map<string, double> window_to_throughput;
+4: optional map<string, double> window_to_complete_latencies_ms;
+5: optional map<string, i64> window_to_acked;
+6: optional map<string, i64> window_to_failed;
```

Comment:
Here too, we cannot renumber the entires or the thrift code will not be compatible with older versions.